HLTrigger/Configuration/python/Tools/confdb.py

```diff
@@ -333,11 +333,9 @@ def addGlobalOptions(self):
 
     self.data += """
 # enable TrigReport, TimeReport and MultiThreading
-%(process)s.options = cms.untracked.PSet(
-    wantSummary = cms.untracked.bool( True ),
-    numberOfThreads = cms.untracked.uint32( 4 ),
-    numberOfStreams = cms.untracked.uint32( 0 ),
-)
+%(process)s.options.wantSummary = True
+%(process)s.options.numberOfThreads = 4
+%(process)s.options.numberOfStreams = 0
```

Comment:
Hmm, that now assumes that ```options``` exists in the menu in ConfDb, so will not work on older menus where there is no ```options``` PSet.

Reply:
I think it should still work (see item 1. in "PR validation"). This is what I meant in the description with (I will try to write it better)

> In recent CMSSW releases (CMSSW_11 and higher), the types of variables inside the options PSet do not need to be specified (even if process.options is not explictly defined in the python config), so those type specifications are dropped.

The following simple test works for me. Basically, `cms.Process` already creates the `options` PSet (at least in `CMSSW >= 11_X_Y`).
```shell
#!/bin/bash

cat <<@EOF > test.py
import FWCore.ParameterSet.Config as cms
process = cms.Process( "HLT" )
process.options.wantSummary = True
@EOF
edmConfigDump test.py > test_dump.py
python3 test.py
python3 test_dump.py
```

Reply:
OK!